mitype/app.py

```diff
@@ -172,6 +172,9 @@ def setup_print(self, win):
         # Print text in BOLD from 3rd line
         win.addstr(2, 0, self.text, curses.A_BOLD)
 
+        # Set cursor position to the start of 3rd line
```

Comment:
This comment can be better written as:
`Set cursor position to beginning of text`
Please change this. :)

Reply:
sure